tests/test_lpass.py

```diff
@@ -4,61 +4,47 @@
 
 
```

Comment:
Please add an impossible-to-miss comment here with the same disclaimer from lpass.py